app/controllers/concerns/redirect_logic.rb

```diff
@@ -3,7 +3,7 @@
 module RedirectLogic
   extend ActiveSupport::Concern
 
-  def redirect_or_404
+  def redirect_or_404(*args)
```

Comment:
What's the use case for the splat here? Looks like you only even have one parmaters passed, the instance object or nil. Just kind of obscures what is happening. And then you have to use `args[0]` below, which is kind of weird.

Reply:
When the controller did not set an instance, an error was thrown in the method when it tried to check the attribute of nil. I needed an optional argument that could also handle nil. Maybe a better way?

Reply:
Maybe `def redirect_or_404(object_or_nil=nil)`? 

Providing a default value means it can handle no params.

Reply:
@cdoyle-temple Or something like that.